charts/ssh/templates/ssh.deployment.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "ssh.fullname" . }}
+  labels:
+    {{- include "ssh.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.ssh.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "ssh.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "ssh.selectorLabels" . | nindent 8 }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          env:
+          - name: RABBITMQ_USERNAME
+            valueFrom:
+              secretKeyRef:
+                key: RABBITMQ_USERNAME
+                name: ssh-env
+          - name: RABBITMQ_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                key: RABBITMQ_PASSWORD
+                name: ssh-env
+          - name: JWTSECRET
+            valueFrom:
+              secretKeyRef:
+                key: JWTSECRET
+                name: ssh-env
+          - name: SERVICE_NAME
```

Comment:
I don't know if this variable and `CRONJOBS` are used - if not, we can just remove them.